apps/condo/domains/resident/utils/helpers.js

```diff
@@ -0,0 +1,68 @@
+const get = require('lodash/get')
+const dayjs = require('dayjs')
+const isSameOrBefore = require('dayjs/plugin/isSameOrBefore')
+const { NOT_FOUND_ERROR } = require('@condo/domains/common/constants/errors')
+
+dayjs.extend(isSameOrBefore)
+
+/**
+ * Connects or disconnects residents to/from organization & property.
+ * Property should be provided to connect.
+ * In other case residents will be disconnected.
+ * @param ResidentAPI (API) either Server or Test API
+ * @param context (context)
+ * @param residents (Resident[])
+ * @param property (Property)
+ * @param reconnectToOlder (boolean) forces reconnecting residents with non-deleted property to restored older one
+ * @returns {Promise<void>}
+ */
+const connectResidents = async (ResidentAPI, context, residents, property, reconnectToOlder = false) => {
```

Comment:
Could you please don’t pass the API as argument. It’s looks like a new code style

Reply:
You don’t need the last argument

Reply:
fixed